docs/performance/storage_usage.md

```diff
@@ -0,0 +1,325 @@
+
+Data was reduced: `18%`, similar to `no_index`.
+
+## Changing the compression algorithm
+
+Data is compressed when it is stored on disk, two options are available `default` (LZ4) and
+`best_compression`.
+
+`best_compression` might be less performant on certain queries, but it has less storage footprint.
+
+You can change it via table definition:
+
+```sql
+CREATE TABLE IF NOT EXISTS "doc"."taxi_january_nocolumnstore" (
+    "VendorID" BIGINT,
+    ...
+) WITH (codec = 'best_compression')
+```
+
+### Effects on storage
+
+| avg_mb_per_shard | total_mb |
+|------------------|----------|
+| 44               | 536      |
+
+Data was reduced: `31.25%`
+
+## All results and what to do.
+
+In the following table, all the above results can be found, also different combinations of them:
+
+```sql
+SELECT table_name,
+       SUM(num_docs)           as records,
+       (SUM(size) / 1_000_000) as total_size_mb,
+       (SUM(size) / count(*)) / 1_000_000 as avg_size_per_shard_in_mb, (SUM(size) / SUM(num_docs) :: DOUBLE) as avg_size_in_bytes_per_record
+FROM sys.shards
+WHERE
+    PRIMARY
+GROUP BY
+    1
+ORDER BY
+    avg_size_per_shard_in_mb
+```
+
+| table_name                                  | records | total_size_mb | avg_size_per_shard_in_mb | avg_bytes_per_record |
+|---------------------------------------------|---------|---------------|--------------------------|----------------------|
+| "taxi_nocolumnstore_noindex_bestcompresion" | 2964624 | 122           | 20                       | 41                   |
+| "taxi_nocolumnstore_bestcompression"        | 2964624 | 205           | 34                       | 69                   |
+| "taxi_noindex_bestcompression"              | 2964624 | 212           | 35                       | 71                   |
+| "taxi_nocolumnstore_noindex"                | 2964624 | 237           | 39                       | 80                   |
+| "taxi_bestcompresion"                       | 2964624 | 290           | 48                       | 98                   |
+| "taxi_noindex"                              | 2964624 | 317           | 52                       | 107                  |
+| "taxi_nocolumnstore"                        | 2964624 | 319           | 53                       | 107                  |
+| "taxi"                                      | 2964624 | 385           | 64                       | 130                  |
+
+To summarize:
+
+1. `total_size_mb` is the sum of the disk used in `mb` of all the primary shards
+2. All primary shards make the full table.
+3. By default, every table will have a maximum of one replica.
+
+The original `775MB` can be calculated as:
+
+`64mb per shard * (6 primary shards + 6 replica shards) = 768`
+
+> The result are slightly off `768 ~= 775` because in this example, decimals are being ignored.
+> The goal is to give you an idea on how tweaking some CrateDB aspect can affect storage, being
+> overly precise to the kilobyte level
+> does not matter too much.
```

Comment:
```suggestion
:::{note}
The result are slightly off `768 ~= 775` because in this example, decimals are being ignored.
The goal is to give you an idea on how tweaking some CrateDB aspect can affect storage.
Being overly precise to the kilobyte level does not matter too much.
:::
```